packages/react-composites/src/composites/CallComposite/components/CallControls.tsx

```diff
@@ -86,8 +97,86 @@ export type CallControlOptions = {
    * @defaultValue true
    */
   screenShareButton?: boolean | { disabled: boolean };
+  /* @conditional-compile-remove-from(stable): custom button injection */
+  /**
+   * Inject custom buttons in the call controls.
+   *
+   * @beta
+   */
+  onFetchCustomButtonProps?: CustomCallControlButtonCallback[];
 };
 
+/* @conditional-compile-remove-from(stable): custom button injection */
+/**
+ * Placement for a custom button injected in the {@link CallControls}.
+ *
+ * 'first': Place the button on the left end (right end in rtl mode).
+ * 'afterCameraButton': Place the button on the right (left in rtl mode) of the camera button.
+ * ... and so on.
+ *
+ * It is an error to place the button in reference to another button that has
+ * been hidden via a {@link CallControlOptions} field.
+ *
+ * Multiple buttons placed in the same position are appended in order.
+ * E.g., if two buttons are placed 'first', they'll both appear on the left end (right end in rtl mode)
+ * in the order provided.
+ *
+ * @beta
+ */
+export type ControlBarButtonPlacement =
```

Comment:
Can this be `CallCompositeControlBarButtonPlacement`? (or something like that but less verbose)
My thinking is the MeetingComposite will want the same/similar API but will have more buttons like chat and people buttons

Reply:
Done. `CustomCallControlButtonPlacement` matches other naming conventions introdued in this PR.